webpack/assets/javascripts/react_app/common/Tour/TourSelectors.js

```diff
@@ -0,0 +1,20 @@
+import { createSelector } from 'reselect';
+import { get } from 'lodash';
+
+export const selectTours = state => state.tours;
+
+export const selectActiveTour = createSelector(
+  selectTours,
+  tours => tours && Object.entries(tours).filter(tour => tour[1].running)[0]
```

Comment:
This seems a little easier to read
```suggestion
  tours => tours && Object.keys(tours).find(tourName => tours[tourName].running)
```